### PR TITLE
Assignment 2 @tnadalini

### PR DESCRIPTION
**Pull request by dpeterson8**
https://gray-mushroom-0185eaa1e.1.azurestaticapps.net

### CODE REVIEW

**Comment by AntonioGomez12** [comment]
Great job overall! I like the side scroll menu which shows the listing of available words. The hamburger menu is not working for me when I click it and the about page should be listed in that menu option. The about page is also blank and perhaps adding a Nuxt-to page may make this fix easier. Other than that great job on the assignment!

**Comment by schuywalker** [comment]
Awesome job! These were the only bullet points where I thought you potentially have work left to do. 

Create unit tests for the validWords method ❌✅
Menu option for an About page ❌✅
Create an About page that has a few sentences about this class project and how great Meg is ❌✅
Make sure this has a back button ❌✅
Add gradient to guesses blocks and keys (with CSS) ❌✅

**Comment by hewrtarkhany** [comment]
looking good especially with giving number of words left

**Comment by mmwoodfo** [comment]
Make sure that you run the command `npm run lintfix` this will autofix formatting and linter errors as well as point out other errors in your code such as the ones below:
image

File: wordle-web/components/list.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        });
+        if(check == true) {
+          newList.push(x) 
+        }
+        else {check = true}
+
+      });
+
+      return newList;
+    }
```
Comment (mmwoodfo) [inline wordle-web/components/list.vue:65]: You may consider putting this in your `wordsService.ts` file.

File: wordle-web/layouts/default.vue [hunk 2]
```diff
@@ -3,10 +3,50 @@
...
+      </router-link>
+          <router-link to="/" class="  no-decoration">
+          <v-icon dark color="primary" x-large> mdi-magnify </v-icon>
       </router-link>
       <v-spacer />
       <settings-dialog />
+      <v-app-bar-nav-icon @click.stop="drawer = !drawer"></v-app-bar-nav-icon>
     </v-app-bar>
+
+      <v-navigation-drawer
```
Comment (mmwoodfo) [inline wordle-web/layouts/default.vue:15]: This looks really nice! I would suggest considering splitting this out into it's own component.

**Pull request closed without merging**